docs/_shared/basic-installation.txt

```diff
@@ -0,0 +1,3 @@
+-  Go to ``http://localhost:8080/``.
+-  Accept ``determined`` as the default username and leave the password empty.
```

Comment:
I like the old:

    Accept the default username of ``determined`` and leave the password empty

Because "accepting `determined` as the default username" sounds like they are actually configuring the default username, which is not what's happening here.

Reply:
resolved

changed all instances to "Accept the default username of ``determined``"